using_images/other_images/jenkins.adoc

```diff
@@ -52,18 +52,96 @@ You can find https://github.com/openshift/origin/tree/master/examples/image-stre
+they should only be required for development testing or atypical configurations accessing a different {product-title} installation for OAuth than the installation hosting Jenkins.
+
+The login process is comprised of a series of HTTP redirects.  Rather than the traditional Jenkins login page, you will first be directed to an {product-title} login page, similar to what you see when you log into the web console.  If login
+is successful, you will be redirected to the page within the Jenkins UI you are trying to access.
+
+Valid user / password combinations are controlled by the identity provider in use with {product-title}.  So, for example,  if the default identity provider `Allow All` is used, you can provide any non-empty string for both the user name
+and password.  If you supply the user name used to create the `*Project*` Jenkins is running in, you will have administration privileges.
+
+If one of the other identity providers is running, valid user/password combinations stored with your identity provider must be used to login.
+
+Valid users are automatically added to the Jenkins authorization matrix as part of login.
+
+The {product-title} `*Role*` of the user then dictates which permissions within the Jenkins authorization matrix are associated with the user.  The `admin` `*Role*` maps to the set of
+permissions traditionally associated with the `admin` user in the {product-title} Jenkins image:
+
+* hudson.model.Computer.Configure
+* hudson.model.Computer.Delete
+* hudson.model.Hudson.Administer
+* hudson.model.Hudson.Read
+* hudson.model.Item.Build
+* hudson.model.Item.Configure
+* hudson.model.Item.Create
+* hudson.model.Item.Delete
+* hudson.model.Item.Read
+* hudson.model.Item.Workspace
+* hudson.model.Run.Delete
+* hudson.model.Run.Update
+* hudson.model.View.Configure
+* hudson.model.View.Create
+* hudson.model.View.Delete
+* hudson.scm.SCM.Tag
+
+Users with the `edit` or `view``*Roles*` for the {product-title} project have progressively reduced permissions within Jenkins.
+
+For the `view` `*Role*`:
+
+* hudson.model.Hudson.Read
+* hudson.model.Item.Read
+
+For the `edit` `*Role*`, in addition to the permissions available to `*view*`:
+
+* hudson.model.Item.Build
+* hudson.model.Item.Configure
+* hudson.model.Item.Create
+* hudson.model.Item.Delete
+* hudson.model.Item.Workspace
+* hudson.scm.SCM.Tag
+
+Users authenticated against {product-title} OAuth will be added to the Jenkins authorization matrix upon their first successful login.
+
+Permissions for users in Jenkins can be changed in {product-title} after those users are initially established in Jenkins.  The OpenShift Login plugin polls the {product-title} API server for permissions and will update the permissions stored in
+Jenkins for each Jenkins user with the permissions retrieved from {product-title}.  Technically speaking, you can change the permissions for a Jenkins user from the Jenkins UI as well, but those changes will be overwritten the next
```

Comment:
change sentence starting w/ "Technically speaking" to:

"If the Jenkins UI is used to update permissions for a Jenkins user, the permission changes will be overwritten the next time the plugin polls {product-title}."